src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -169,7 +169,7 @@ public void TestExitTime()
             Assert.Throws<InvalidOperationException>(() => p.ExitTime);
             p.Kill();
             Assert.True(p.WaitForExit(WaitInMS));
-            Assert.True(p.ExitTime.ToUniversalTime() >= timeBeforeProcessStart, "TestExitTime is incorrect.");
+            Assert.True(p.ExitTime.ToUniversalTime() >= timeBeforeProcessStart, string.Format("TestExitTime is incorrect. TimeBeforeStart={0}, ExitTime={1}, ExitTimeUniversal={2}", timeBeforeProcessStart, p.ExitTime, p.ExitTime.ToUniversalTime()));
```

Comment:
Nit: things like this can be simplified with string interpolation, e.g.
```C#
Assert.True(p.ExitTime.ToUniversalTime() >= timeBeforeProcessStart, $"TestExitTime is incorrect. TimeBeforeStart={timeBeforeProcessStart}, ExitTime={p.ExitTime}, ExitTimeUniversal={p.ExitTime.ToUniversalTime()}"
```